packages/common/src/types/semanticLayer.ts

```diff
@@ -222,3 +222,23 @@ export function getFilterFieldNamesRecursively(filter: SemanticLayerFilter): {
         ...orFiltersFieldNames,
     ];
 }
+
+// Helper functions to convert between the column names in the query and the column names in the results
+
+export function convertColumnToResultsColumn(
+    column: string,
+    resultsColumns: string[],
+) {
+    return resultsColumns.find(
+        (resultCol) => resultCol.toLowerCase() === column.toLowerCase(),
```

Comment:
So the results column names arent case sensitive?

Reply:
Good question, I don't think so but perhaps this is something we should look into a bit better, but from my understanding the semantic layer only needs to receive the fields and they do that mapping to do the query on the warehouse, but then the results given to us are not mapped with column name === field name, so we kinda need to do it again on our side? 🤔